src/components/views/dialogs/ConfirmCloseHostingSignupDialog.js

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { _t } from "../../../languageHandler";
+import * as sdk from '../../../index';
+
+/*
+ * A dialog for confirming closing the Hosting signup setup dialog.
+ */
+export default class ConfirmCloseHostingSignupDialog extends React.Component {
+    render() {
+        const QuestionDialog = sdk.getComponent('views.dialogs.QuestionDialog');
```

Comment:
`import` instead of `getComponent` please

Reply:
Is there a reason `getComponent` seems to be used throughout the codebase a lot - legacy?

Reply:
Yup, legacy, it is very seldom used in Typescript code, especially new code.